Documentation/ExtensionArchitecture/FileStructure/Configuration/Index.rst

```diff
@@ -23,7 +23,7 @@ folder:
 .. code-block:: none
    :caption: Common content of the configuration folder
 
-   $ tree public/typo3conf/ext/my_extension/Configuration/TypoScript/
+   $ tree vendor/myvendor/my-extension/Configuration/
```

Comment:
We should stick to "packages/local_packages". We mix two things up: storing (by a developer) and installed (by Composer). Also examples below.